### ISSUE TITLE
menu_create multi language param not work

### ISSUE THREAD

**Issue by SmileYzn** [alliedmodders/amxmodx#880]
# Help us help you
  - [x] I have checked that my issue [doesn't exist yet](https://github.com/alliedmodders/amxmodx/issues).
  - [x] I have tried my absolute best to reduce the problem-space and have provided the absolute smallest test-case possible.
  - [x] I can always reproduce the issue with the provided description below.

# Environment
  * Operating System version: 
  * Game/AppID (with version if applicable): 
  * Current AMX Mod X version: 
  * Current Metamod version:
  - [x] I have updated AMX Mod X to the [latest version](https://www.amxmodx.org/downloads.php) and it still happens.
  - [x] I have updated AMX Mod X to the [latest snapshot](https://www.amxmodx.org/snapshots.php) and it still happens.
  - [x] I have updated Metamod to the [latest version](https://www.amxmodx.org/downloads.php) and it still happens.

# Description
menu_create multi language param not work.
I used test code from here: https://github.com/alliedmodders/amxmodx/pull/593#issue-217270055

Someone can confirm? I'm using latest amxmodx 1.10

# Problematic Code (or Steps to Reproduce)
  ```PAWN
#include <amxmodx>

new StaticMenuHandle;

public plugin_init()
{
    buildStaticMenu();

    register_dictionary("menutest.txt");
    register_clcmd("menu", "@OnCommandDisplayMenu");
}

public plugin_end()
{
    StaticMenuHandle && menu_destroy(StaticMenuHandle);
}

@OnCommandDisplayMenu(const player)
{
    menu_display(player, StaticMenuHandle);

    return PLUGIN_HANDLED;
}

buildStaticMenu()
{
    new const menu = menu_create("MENU_NAME", "@OnStaticMenu", .ml = true);

    menu_additem(menu, "ITEM_1");
    menu_additem(menu, "ITEM_2");
    menu_additem(menu, "ITEM_3");
    menu_additem(menu, "ITEM_4");

    StaticMenuHandle = menu;
}

@OnStaticMenu(const player, const menu, const item)
{

}
  ```

# Logs
No logs are given

**Comment by afwn90cj93201nixr2e1re** [comment]
![изображение](https://user-images.githubusercontent.com/9306996/87343036-c52ca080-c53b-11ea-88bd-a7bac8c1cc28.png)


```
#include <amxmodx>

new StaticMenuHandle = -1;

public plugin_init()
{
  buildStaticMenu();

  register_dictionary("menutest.txt");
  register_clcmd("say 4", "OnCommandDisplayMenu");
}

public plugin_end()
{
  StaticMenuHandle != -1 && menu_destroy(StaticMenuHandle);
}

public OnCommandDisplayMenu(const player)
{
  menu_display(player, StaticMenuHandle);

  return PLUGIN_HANDLED;
}

buildStaticMenu()
{
  StaticMenuHandle = menu_create("MENU_NAME", "OnStaticMenu", .ml = true);

  menu_additem(StaticMenuHandle, "ITEM_1");
  menu_additem(StaticMenuHandle, "ITEM_2");
  menu_additem(StaticMenuHandle, "ITEM_3");
  menu_additem(StaticMenuHandle, "ITEM_4");
}

public OnStaticMenu(const player, const menu, const item)
{

}
```

**Comment by Arkshine** [comment]
That you see the translation in-game means it works. 
If you say that the language is incorrect, then check your language setting.
Or did I miss something?

**Comment by afwn90cj93201nixr2e1re** [comment]
No, u r'nt, but possible he just disabled multilang.amxx or smthng like that, btw he can just enable amx_mldebug for more info.

**Comment by SmileYzn** [comment]
I just uses directly given plugin without others, and what Multi-Lingual System plugin do in amxx lang system? nothing i think

@Arkshine Just not works at all.

**Comment by afwn90cj93201nixr2e1re** [comment]
Show ur amxx list output.
Ofc u can leave multilang too, 
And meta cvars output.

**Comment by afwn90cj93201nixr2e1re** [comment]
Are u sure about lang file exists? It's not default lang file. U should create it. I have no problems with given code on latest amxmodx.

**Comment by SmileYzn** [comment]
Already fixed it, i has replaced amxx 1.9 dll into amxx 1.10 install.

Ps. I thought that amxx 1.9 already have translation into menus, but no!

**Comment by afwn90cj93201nixr2e1re** [comment]
No, ark. don't wanna merge it to 1.9 coz it's new feature, 1.9 == 1.8.3 for hin.

**Comment by SmileYzn** [comment]
Did you read the comment? I had replaced accidentally the 1.10 DLL  by 1.90.

**Comment by afwn90cj93201nixr2e1re** [comment]
So, and what's wrong with my comment? I just described why it's was not implemented in 1.9.
